File: python3/2418.py
class Solution:
    def sortPeople(self, names: List[str], heights: List[int]) -> List[str]:
        people = []
        for i in range(len(names)):
            people.append((heights[i], names[i]))
        people = sorted(people, reverse=True)
        for i in range(len(people)):
            people[i] = people[i][1]
        return people

File: golang/2418.go
func sortPeople(names []string, heights []int) []string {
    sorted := false
    for !sorted {
        sorted = true
        for i := 0; i < len(names) - 1; i++ {
            if heights[i] < heights[i + 1] {
                sorted = false
                names[i], names[i + 1] = names[i + 1], names[i]
                heights[i], heights[i + 1] = heights[i + 1], heights[i]
            }
        }
    } 
    return names
}
